variables.tf

```diff
@@ -66,6 +66,79 @@ variable "app_config_collections" {
   default = []
 }
 
+##############################################################
+# Configuration aggregator
+##############################################################
+
+variable "enable_config_aggregator" {
```

Comment:
Need to update the usage section in the readme to include the new variables added

Reply:
The `enable_config_aggregato`r is disabled by default. Its not even used for App Config use cases. Its only needed by workload protection, so I purposely didn't change the usage in the App Config readme (if thats what you were asking)